packages/components/src/components/select/select.ts

```diff
@@ -223,6 +224,8 @@ export default class SdSelect extends SolidElement implements SolidFormControl {
         removable
         @keydown=${(event: KeyboardEvent) => this.handleTagKeyDown(event, option)}
         @sd-remove=${(event: CustomEvent) => this.handleTagRemove(event, option)}
+        @mouseenter=${this.handleMouseEnter}
```

Comment:
I just realized, that the select feels really laggy. I saw, that the hover styles are triggered by JavaScript, instead of `hover:`. Could we refactor that in this PR? (Maybe ahs to be changed in Combobox as well.)

Reply:
Updated both `sd-select` and `sd-combobox`.
Hover was being applied with js because of some issue in Safari but I could not replicate so the switch should be ok.

I think the component is laggy in storybook because it has many stories. I tested by leaving just the default and it felt much better.